packages/bpk-component-floating-notification/src/accessibility-test.tsx

```diff
@@ -30,7 +31,7 @@ const props = {
 
 describe('BpkFloatingNotification accessibility tests', () => {
   it('should not have programmatically-detectable accessibility issues', async () => {
-    const { container } = render(<BpkFloatingNotification />);
+    const { container } = render(<BpkFloatingNotification {...props} />);
```

Comment:
Just want to clarify the addition of the `...props` here?

Reply:
The `text` prop is required in the component, calling it without it would fail. 
I added the whole `props` object since it only has the text property.